data/brands/amenity/bank.json

```diff
@@ -10140,7 +10140,11 @@
         "brand:wikidata": "Q4340839",
         "brand:wikipedia": "uk:Ощадбанк",
         "name": "Ощадбанк",
-        "name:en": "State Savings Bank of Ukraine"
+        "name:en": "Oschadbank",
```

Comment:
1. why not change `brand:en` too?
2. If only have "*name"+"*name:en" but without "*name:local", many program will prefer the English version, see: https://wiki.openstreetmap.org/wiki/Multilingual_names#Repeating_name_with_language_specific_tag

Reply:
Thanks for the remarks it's fixed now.